mail-storage/src/main/java/mail/storage/util/MessageUtils.java

```diff
@@ -0,0 +1,34 @@
+package mail.storage.util;
+
+import mail.storage.domain.Message;
+import mail.storage.dto.MessageDto;
+import mail.storage.dto.UpdateMessageDto;
+
+import java.util.Date;
+
+public class MessageUtils {
```

Comment:
it's not utils logic it has to be encapsulated in Message

Reply:
Fixed.